### PR TITLE
Show profile

### PR DESCRIPTION
**Pull request by yongsokheng**

### CODE REVIEW

File: app/controllers/profiles_controller.rb [hunk 1]
```diff
@@ -1,17 +1,27 @@
 class ProfilesController < ApplicationController
-  before_action :load_data, only: [:edit, :update]
+  before_action :load_user
+  before_action :load_data, except: :index
+
+  def index
+    unless @user.recruiter?
```
Comment (NgengChhengkim) [inline app/controllers/profiles_controller.rb]: `if @user.applicant?` ?

File: app/controllers/profiles_controller.rb [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
+      @skills = @user.skills
+      @experiences = @user.experiences.order_date
```
Comment (NgengChhengkim) [inline app/controllers/profiles_controller.rb]: @user.experience=nil.  Error ko?

File: app/views/profiles/_company_profile_detail.html.erb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<div id="company-profiles">
+  <table>
+    <tr>
+      <td class="title"><%= t "profiles.website" %></td>
+      <td><%= link_to @user.company_profile_website, @user.company_profile.website,
```
Comment (NgengChhengkim) [inline app/views/profiles/_company_profile_detail.html.erb]: Delegate

File: app/views/profiles/_company_profile_detail.html.erb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+      target: "_blank" if @user.company_profile %></td>
+    </tr>
+    <tr>
+      <td class="title"><%= t "profiles.start_up_date" %></td>
+      <td><%= @user.company_profile_start_up_date if @user.user_profile if @user.company_profile %></td>
```
Comment (NgengChhengkim) [inline app/views/profiles/_company_profile_detail.html.erb]: Sao lai `if` 2 lan?

File: app/views/profiles/_education_detail.html.erb [hunk 5]
```diff
@@ -0,0 +1,19 @@
+
```
Comment (NgengChhengkim) [inline app/views/profiles/_education_detail.html.erb]: Xoa di

File: app/views/profiles/_education_detail.html.erb [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+<div class="school"><%= education.school %></div>
+<div>
+  <%= t "profiles.from" %>
+  <%= "#{l education.start_date, format: :default} #{t "profiles.labels.to"}
+      #{l education.end_date, format: :default}" %>
+</div>
+<div>
+  <%= t "profiles.degree" %>
+  <%= education.degree.name %>
```
Comment (NgengChhengkim) [inline app/views/profiles/_education_detail.html.erb]: Delegate

File: app/views/profiles/_user_profile_detail.html.erb [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+      <td class="title"><%= t "profiles.current_position" %></td>
+      <td><%= @user.user_profile_current_position if @user.user_profile %></td>
+    </tr>
+    <tr>
+      <td class="title"><%= t "profiles.birth_date" %></td>
+      <td><%= @user.user_profile_birth_date if @user.user_profile %></td>
+    </tr>
+    <tr>
+      <td class="title"><%= t "profiles.gender" %></td>
+      <td><%= @user.user_profile_gender if @user.user_profile %></td>
```
Comment (NgengChhengkim) [inline app/views/profiles/_user_profile_detail.html.erb]: Kiem tra `if` Chung mot lan va kiem tra 3 lan cai nao hay hon?  Neu ko co hien thi text neu co hieun thi ca text ca gia tri

File: app/controllers/profiles_controller.rb (hunk 1 above)
Comment (yongsokheng) [reply app/controllers/profiles_controller.rb]: khac recruiter, co the la admin / applicant

File: app/views/profiles/_company_profile_detail.html.erb (hunk 4 above)
Comment (yongsokheng) [reply app/views/profiles/_company_profile_detail.html.erb]: nham :D

File: app/views/profiles/_user_profile_detail.html.erb (hunk 7 above)
Comment (yongsokheng) [reply app/views/profiles/_user_profile_detail.html.erb]: Y la khi ko co user_profile cung hien thi text de cho nguoi xem biet la co nhung thong tin nao. nhung thong tin day chua nhap

**Pull request merged**